src/pages/overview/pdf-services-api/howtos/combine-pdf.md

```diff
@@ -85,54 +85,89 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
 // cd CombinePDF/
 // dotnet run CombinePDF.csproj
 
- namespace CombinePDF
- {
-   class Program
-   {
-     private static readonly ILog log = LogManager.GetLogger(typeof(Program));
-     static void Main()
-     {
-       //Configure the logging
-       ConfigureLogging();
-       try
-       {
-           // Initial setup, create credentials instance.
-           Credentials credentials = Credentials.ServicePrincipalCredentialsBuilder()
-                .WithClientId("PDF_SERVICES_CLIENT_ID")
-                .WithClientSecret("PDF_SERVICES_CLIENT_SECRET")
-                .Build();
-
-         //Create an ExecutionContext using credentials and create a new operation instance.
-         ExecutionContext executionContext = ExecutionContext.Create(credentials);
-         CombineFilesOperation combineFilesOperation = CombineFilesOperation.CreateNew();
-
-         // Add operation input from source files.
-         FileRef combineSource1 = FileRef.CreateFromLocalFile(@"combineFilesInput1.pdf");
-         FileRef combineSource2 = FileRef.CreateFromLocalFile(@"combineFilesInput2.pdf");
-         combineFilesOperation.AddInput(combineSource1);
-         combineFilesOperation.AddInput(combineSource2);
-
-         // Execute the operation.
-         FileRef result = combineFilesOperation.Execute(executionContext);
-
-         // Save the result to the specified location.
-         result.SaveAs(Directory.GetCurrentDirectory() + "/output/combineFilesOutput.pdf");
-
-       }
-       catch (ServiceUsageException ex)
-       {
-         log.Error("Exception encountered while executing operation", ex);
-       }
-       // Catch more errors here. . .
-     }
+namespace CombinePDF
+{
```

Comment:
Not related to this file, please add .NET code snippets in 

pdf-services-api/howtos/sdk-utilities/

Reply:
Fixed